app.rb

```diff
@@ -162,21 +162,21 @@
     # Vital Signs Searching
     # Vital Signs includes these codes as defined in http://loinc.org
     vital_signs = {
-      '9279-1' => 'Respiratory rate',
-      '8867-4' => 'Heart rate',
-      '2710-2' => 'Oxygen saturation in Capillary blood by Oximetry',
-      '55284-4' => 'Blood pressure systolic and diastolic',
-      '8480-6' => 'Systolic blood pressure',
-      '8462-4' => 'Diastolic blood pressure',
-      '8310-5' => 'Body temperature',
-      '8302-2' => 'Body height',
-      '8306-3' => 'Body height --lying',
-      '8287-5' => 'Head Occipital-frontal circumference by Tape measure',
-      '3141-9' => 'Body weight Measured',
-      '39156-5' => 'Body mass index (BMI) [Ratio]',
-      '3140-1' => 'Body surface area Derived from formula',
-      '59408-5' => 'Oxygen saturation in Arterial blood by Pulse oximetry',
-      '8478-0' => 'Mean blood pressure'
+      '9279-1' => 'Respiratory Rate',
```

Comment:
You shouldn't change the capitalization of these `vital_signs` -- those are the "official" descriptions. Terminology folks can get crazed about making sure the "display" or "description" matches the official code description.